### ISSUE TITLE
Test fails for Set-vRAUserPrincipal v7.1 

### ISSUE THREAD

**Issue by chelnak** [jakkulabs/PowervRA#83]
Updating a user seems to fail for all properties except Password.

The interesting thing is that if you update Password plus any of the other available properties it works.

I think they must have changed something with the validation of the api. Would be worth testing this against 7.2.

**Fail:**
```
Set-vRAUserPrincipal -Id testuser@vsphere.local -FirstName "Hello"
```
**Pass:**
```
Set-vRAUserPrincipal -Id testuser@vsphere.local -FirstName "Hello" -Password New!password01
```
**Error:**
```
{
  "errors": [
    {
      "code": 50505,
      "message": "System exception.",
      "moreInfoUrl": null
    }
  ]
}
```

**Comment by jonathanmedd** [comment]
Track it on the communities here:
https://communities.vmware.com/message/2633805#2633805

**Comment by jonathanmedd** [comment]
This still seems to be a problem. Despite the fix that was implemented, the test is still failing on 7.1.....

Set-vRAUserPrincipal -Id $JSON.Principal.UserPrincipalId -FirstName $JSON.Principal.UserPrincipalFirstNameUpdated -Password $Password

causes a failure:

Invoke-RestMethod : Response status code does not indicate success: 400 (Bad Request).

Running the following, doesn't have a problem:

Set-vRAUserPrincipal -Id $JSON.Principal.UserPrincipalId -FirstName $JSON.Principal.UserPrincipalFirstNameUpdated

**Comment by jonathanmedd** [comment]
Scrap this, my mistake. Was running an old version of the test. Running the correct test version and all is good. Plus have also run some independent tests to confirm it.
